fern/pages/sdks/introduction/customer-showcase.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Customer Showcase
+---
+
+At Fern, we're proud to enable businesses to build and distribute world-class SDKs. From startups to industry leaders, our customers use Fern to streamline their development processes and enhance their developer experience. Below are just a few examples of companies that use Fern's capabilities to build and deploy their SDKs.  
+
+---
+
+## **Square**  
+Square trusts Fern to generate and maintain **7 SDKs**, ensuring their APIs are easily accessible to developers building on any platform.  
```

Comment:
can we add a badge after the company name that has the feature labeled, such as `REST`, `gRPC`, or `file upload`